lua/astrocommunity/editing-support/codecompanion-nvim/README.md

```diff
@@ -0,0 +1,11 @@
+# Codecompanion.nvim
```

Comment:
Would be nice to have a bit more info about the plugin :slightly_smiling_face: 
Now it's very barebones